src/components/common/BottomButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode, useState } from "react";
+import { styled } from "styled-components";
+
+type BottomButtonProps = {
+  children: ReactNode;
+  isactive: boolean;
+};
+
+export default function BottomButton({ children, isactive }: BottomButtonProps) {
+  return (
+    <BottomContainer isactive={isactive}>
```

Comment:
```suggestion
    <BottomContainer $isactive={isactive}>
```
transient props $ 이용해주세요!

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ ㅎ 코드리뷰의 중요성 죄송합니다 죄송합니다